feathr_project/feathr/_feature_registry.py

```diff
@@ -838,3 +839,142 @@ def _list_registered_entities_with_details(self, project_name: str = None, entit
         entity_res = [] if guid_list is None else self.purview_client.get_entity(
             guid=guid_list)["entities"]
         return entity_res
+        
+    def get_features_from_registry(self, project_name: str) -> Tuple[List[FeatureAnchor], List[DerivedFeature]]:
+        """[Sync Features from registry to local workspace, given a project_name, will write project's features from registry to to user's local workspace]
+        Args:
```

Comment:
let's give users a documentation warning here that if the project is big, the return result is huge.

Reply:
Added